alea/include/alps/alea/autocorr.hpp

```diff
@@ -199,6 +199,10 @@ class autocorr_result
 
     level_result_type &level(size_t i) { return level_[i]; }
 
+    /** Check if this result is identical to another */
+    bool operator==(const autocorr_result &other) const;
+    bool operator!=(const autocorr_result &other) const { return !operator==(other); }
```

Comment:
A little bit of bikeshedding: I find non-inplace binary operators are conceptually clearer and more versatile as free functions.  Not a breaker though.
See also: http://www.drdobbs.com/cpp/how-non-member-functions-improve-encapsu/184401197